src/main/java/com/kodilla/ecommercee/controller/ProductController.java

```diff
@@ -16,32 +17,35 @@
 @RequiredArgsConstructor
 public class ProductController {
 
+    @Autowired
```

Comment:
sugestia: wstrzykujcie zależności przez konstruktor, https://docs.spring.io/spring-boot/docs/2.0.x/reference/html/using-boot-spring-beans-and-dependency-injection.html